launchlib/config.go

```diff
@@ -71,7 +71,9 @@ type CustomLauncherConfig struct {
 	DisableContainerSupport bool                       `yaml:"dangerousDisableContainerSupport"`
 }
 
-type ExperimentalLauncherConfig struct{}
+type ExperimentalLauncherConfig struct {
+	ExperimentalContainerV2 bool `yaml:"experimentalContainerV2"`
```

Comment:
Perhaps we call this `containerV2` since the parent section is called `experimental` (sorry I originally suggested including experimental in the prop name!)

Reply:
Changed. Thought you were being extra explicit that this was experimental! 😄